src/statsd_client.cpp

```diff
@@ -88,25 +115,21 @@ int StatsdClient::init()
     d->server.sin_family = AF_INET;
     d->server.sin_port = htons(d->port);
 
-    int ret = inet_aton(d->host.c_str(), &d->server.sin_addr);
```

Comment:
any particular reason for removing this? doesn't that take away the ability to pass an IP address?